sdk/resourcemanager/internal/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/internal
+
+go 1.16
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.22.0
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.13.1
+	github.com/Azure/azure-sdk-for-go/sdk/internal v0.9.1
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v0.3.1
```

Comment:
Would this be a cyclic reference between this module and resources module?

Reply:
I notice the cyclic reference when add this dep. But I've tested the `testutil` in `armresources`, it seems no mod import problem.

Reply:
Importing these does not have problems. But we might have problems when we have to update these two. But this is a minor problem

Reply:
Got it. We may carefully handle it by specific release step.